contrib/opentimelineio_contrib/adapters/ale.py

```diff
@@ -324,7 +324,7 @@ def write_to_string(input_otio, columns=None, fps=None, video_format=None):
                     columns.append(key)
 
     # Always output these
-    for c in ["Duration", "End", "Start", "Name", "Source File"]:
+    for c in ["Duration", "End", "Start", "Name", "Source File", "ASC_SOP", "ASC_SAT"]:
```

Comment:
I think we may not want to add `ASC_SOP` and `ASC_SAT` here - we only want to include those columns if one of the clips uses them.